src/Layers/xrRenderGL/glSH_Texture.cpp

```diff
@@ -27,8 +27,8 @@ void resptrcode_texture::create(LPCSTR _name)
 //////////////////////////////////////////////////////////////////////
 CTexture::CTexture()
 {
-    pSurface = NULL;
-    pBuffer = NULL;
+    pSurface = 0;
```

Comment:
nullptr? Or it is not a pointer? So it's a strange naming. And I suggest you to use initializer list (or even initialize such members directly in the class definition)

Reply:
It would be better to define in renders a correspondence constant and use it to init. But considering quality of the engine I wouldn't touch it at all and let it be as it is.

Reply:
Yes, we'll leave it as TODO.